### PR TITLE
Replaced WinForms-UI with modern WPF-UI!

### PR DESCRIPTION
**Pull request by TRoskop**
Hallo @Nummer ,
as I said before in https://github.com/Nummer/Destroy-Windows-10-Spying/issues/88 , the winforms-UI is not good for newer devices.
So I tried to migrate the UI to WPF.

And here is it. I migrated everything almost 1:1 converting older classes to newer.
Have some fun with it! And you can simly customize it!!

### CODE REVIEW

File: DWS/DWS_Lite.csproj [hunk 1]
```diff
@@ -9,12 +9,13 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>DWS_Lite</RootNamespace>
     <AssemblyName>DWS_Lite</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
+    <ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
```
Comment (TRoskop) [inline DWS/DWS_Lite.csproj]: The new GUID {60d..... informs VS that we are using WPF.

File: DWS/DWS_Lite.csproj [hunk 2]
```diff
@@ -32,6 +34,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Prefer32Bit>false</Prefer32Bit>
```
Comment (TRoskop) [inline DWS/DWS_Lite.csproj]: Mosts Windows 10 devices supports 64-bit

**Comment by DanWillman** [comment]
@TRoskop, I don't do much with WPF right now but have been looking at it. Do you just use the usual gui tools or did you build in Blend? Just curious

**Comment by TRoskop** [comment]
@DanWillman ,
Well, I use VS 2015 Community and its build-in designer. But mostly I code XAML by hand because it is for me much easier.

**Comment by DanWillman** [comment]
@TRoskop Cool, thanks much. I've tried a few XAML ui's at work but just go back to using winforms because I find it quicker. I'll have to play around with doing XAML by hand instead of using the designer though, I just stumbled across Blend and thought maybe that's what most use

**Comment by Nummer** [comment]
THX !! COOLLL!! ADD!!!

**Comment by Nummer** [comment]
language "combobox" not found :D

**Comment by TRoskop** [comment]
Yes, this is because according to MSDN (see https://msdn.microsoft.com/de-de/library/ms788718%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396) WPF decides which language to use (normaly it is the os's language). So it is not common that the user can change the language.
But if you want, I can add it of course.

**Comment by TRoskop** [comment]
Well, should I merge now?

**Comment by Nummer** [comment]
sry, no

**Pull request closed without merging**